metasploit_payloads-mettle.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |spec|
   spec.license       = '3-clause (or "modified") BSD'
 
   spec.files         = `git ls-files lib/`.split("\n")
-  spec.files        += Dir['build/*/bin/*']
+  spec.files        += Dir['build/*/bin/*'].reject { |file| file =~ /^build\/tools/ }
```

Comment:
This should be `file.start_with? "build/tools"`

Reply:
Sure, thought the same change could apply to metasploit-framework.gemspec

Reply:
(well, maybe, it checks for multiple things)

Reply:
Fixed, thanks